README.md

```diff
@@ -1,2 +1,27 @@
 # Hoodoo-sample
 A project for learning some key aspects of the Hoodoo framework.
+
```

Comment:
Could of minor formatting things - visible when you [preview the page](https://github.com/KevinWeirNZ/Hoodoo-sample/tree/feature/ft-793-time-service):

<img width="1165" alt="image" src="https://user-images.githubusercontent.com/636717/142476160-b74d2d5d-b021-4eab-a172-a2ea551e5e58.png">